java/src/jmri/jmrit/beantable/LogixTableAction.java

```diff
@@ -564,6 +659,199 @@ protected String helpTarget() {
     private boolean _newItem = false; // marks a new Action or Variable object was added
 
     /**
+     * Input selection names
+     */ 
+    public enum SelectionMode {
+        USEMULTI, USESINGLE, USECOMBO;
+    }
+    SelectionMode _selectionMode;
+    
+    // Retain the menu objects for the doClick() method.
+    JRadioButtonMenuItem menuMulti; 
+    JRadioButtonMenuItem menuSingle; 
+    JRadioButtonMenuItem menuCombo; 
+     
+    /**
```

Comment:
I thought that was intended as a comment on the enum to describe that this enum is a set of names for input types.

Javadoc handles both comments on the entire enum, like this one, and also individual elements. See [line 110 of VerifyWriteProgrammerFacade](https://github.com/JMRI/JMRI/blob/master/java/src/jmri/implementation/VerifyWriteProgrammerFacade.java#L110) for an example of how to write it. It then shows up both in the [class Javadocs](http://jmri.org/JavaDoc/doc/jmri/implementation/VerifyWriteProgrammerFacade.html#nested.class.summary) and the [separate Javadoc for the enum](http://jmri.org/JavaDoc/doc/jmri/implementation/VerifyWriteProgrammerFacade.ProgState.html).  (The diagram was created and inserted with PlantUML; it's the text part that shows how the Javadoc works)